plexpy/notifiers.py

```diff
@@ -2721,3 +2748,71 @@ def return_config_options(self):
                          ]
 
         return config_option
+
+class HIPCHAT(object):
+
+    def __init__(self):
+        self.apiurl = plexpy.CONFIG.HIPCHAT_URL
+        self.color = plexpy.CONFIG.HIPCHAT_COLOR
+
+    def notify(self, message, subject):
+        if not message or not subject:
+            return
+        
+        text = '(plex) ' + subject.encode('utf-8') + ': ' + message.encode('utf-8')
```

Comment:
Added the subject option.  Thanks for catching the (plex) emoticon tag, I threw that in early and forgot I hadn't even made it optional without any explanations.  I added an option to control that also.

Hipchat lets you define custom emoticons and the messages look nice with a plex emoticon defined and used at the start of the lines like this:
![message_example](https://cloud.githubusercontent.com/assets/605448/16689325/67ef85ba-44f0-11e6-90a7-9e0979b00a70.png)